### ISSUE TITLE
mermaid diagram not rendering

### ISSUE THREAD

**Issue by shanthisagar-dhanya-kumar-db** [johanneswuerbach/backstage-plugin-techdocs-addon-mermaid#2]
I have performed all the steps specified, but the mermaid diagram doesn't render.
The content is displayed as is.

Is there anything I need to add in the mkdocs.yml?

**Comment by shanthisagar-dhanya-kumar-db** [comment]
ok, thanks for clarifying. 
But, then would the typescript issue be taken care in 0.1.1?

**Comment by johanneswuerbach** [comment]
Which version of backstage are you using? The docs addon framework is only available in v1.2.0, which was released yesterday.

**Comment by shanthisagar-dhanya-kumar-db** [comment]
Yes, I am using the addon framework which comes with v1.2.0

**Comment by dmckernanacv** [comment]
Same, using 1.2.0.
I believe the typescript issue @shanthisagar-dhanya-kumar-db is referring to is this one:

```shell
packages/app/src/App.tsx:94:10 - error TS2604: JSX element type 'Mermaid' does not have any construct or call signatures.

94         <Mermaid />
            ~~~~~~~
```

When running `tsc`.

**Comment by johanneswuerbach** [comment]
This should be resolved in version `v0.2.1` of this plugin, which version are you using?

**Comment by dmckernanacv** [comment]
Sorry, yes, it is fixed in v0.2.1, however mermaid is not rendering because of [this code](https://github.com/johanneswuerbach/backstage-plugin-techdocs-addon-mermaid/blob/944c04b316e798d283adf4a88cd03481669d351a/src/Mermaid/Mermaid.tsx#L39)

That class is not being added to the code blocks in backstage 1.2.0.

**Comment by johanneswuerbach** [comment]
Removed in https://github.com/johanneswuerbach/backstage-plugin-techdocs-addon-mermaid/pull/4 and published as `v0.2.2`. Could you give it a try?

**Comment by johanneswuerbach** [comment]
Yes, I totally didn't test that :-(. I've poked a bit around, but couldn't find a way to use addons there. 

Asked in the main thread https://github.com/backstage/backstage/issues/4123#issuecomment-1130464174

**Comment by peimanja** [comment]
Works great in both scenarios. Thanks for the addon !

**Comment by johanneswuerbach** [comment]
Closing as this is resolved. Please open a new issue if you discover additional issues :-)
